docs/about.md

```diff
@@ -55,13 +55,50 @@ Contributor [2021 January - ]<br/>
 
 <!-- ------------------------------------------------------------------------------------------------------ -->
 
-### [Gokul Rajiv](https://github.com/gok99)
+### [Zhou Jiahao](https://github.com/zhoukerrr)
 **Role**: Contributor [2022 January - ]<br/>
```

Comment:
Add "Committer [2023 January - ]" for Zhou Jiahao, Chan Jun Da, Tay Yi Hsuen
```suggestion
**Role**: Committer [2023 January - ]<br/>
Contributor [2022 January - ]<br/>
```

Reply:
Sure thing!